parser/src/parser/file.rs

```diff
@@ -359,6 +373,31 @@ impl ParserContext {
         Ok(members)
     }
 
+    ///
+    /// Returns a [`CircuitMember`] AST node if the next tokens represent a circuit member static constant.
+    ///
+    pub fn parse_const_member_variable_declaration(&mut self) -> Result<(CircuitMember, bool, bool)> {
+        self.expect(Token::Static)?;
+        self.expect(Token::Const)?;
+
+        let name = self.expect_ident()?;
+        self.expect(Token::Colon)?;
+        let type_ = self.parse_type()?.0;
```

Comment:
Repeated in `parse_member_variable_declaration` -- could refactor into a helper method.